packages/core/src/styles/components/_callouts.scss

```diff
@@ -0,0 +1,3 @@
+g.callouts {
```

Comment:
Just want to make sure this is namespaced well enough

Reply:
fixed it!